pkg/commands/cloud_config_import.go

```diff
@@ -845,6 +871,36 @@ func (ctx *executionContext) executeTask(task *testTask, clusterConfigs []string
 		ctx.Lock()
 		for _, inst := range instances {
 			inst.taskCancel = cancel
+			if task != nil && task.test != nil && inst.runningExecution != task.test.ExecutionConfig {
+				if inst.runningExecution != nil {
+					for _, cfg := range clusterConfigs {
+						err = ctx.handleScript(&runScriptArgs{
+							Name:          "After",
+							ClusterTaskId: task.clusterTaskID,
+							Script:        inst.runningExecution.After,
+							Env:           append(inst.runningExecution.Env, fmt.Sprintf("KUBECONFIG=%v", cfg)),
```

Comment:
I think we need to pass Cloud config variable as parameter, to less depend on kubernetes.

Reply:
Can we solve this problem within issue https://github.com/networkservicemesh/cloudtest/issues/1 ?

Reply:
ok